draft-sharabayko-mops-srt.md

```diff
@@ -1100,6 +1103,42 @@ Control Information Field (CIF).
 : A single value or a range of lost packets sequence numbers. See packet sequence number
 coding in {{packet-seq-list-coding}}.
 
+### Congestion Warning {#ctrl-pkt-congestion}
+
+The Congestion Warning control packet is not in use at the moment.
+If the receiver receives this packet, it slows down sending rate
+increasing the minimum inter packet sending interval by 12.5%.
```

Comment:
I think it should be explained why `12.5%` is, but not for this commit.

Reply:
Magic number.